composer.json

```diff
@@ -54,14 +54,14 @@
         }
     },
     "scripts": {
-        "check-cs": "vendor/bin/ecs check --ansi",
-        "fix-cs": "vendor/bin/ecs check --fix --ansi",
-        "phpstan": "vendor/bin/phpstan analyse --ansi --error-format symplify",
-        "rector": "vendor/bin/rector process --config rector-ci.yaml",
+        "check-cs": "\"vendor/bin/ecs\" check --ansi",
+        "fix-cs": "\"vendor/bin/ecs\" check --fix --ansi",
+        "phpstan": "\"vendor/bin/phpstan\" analyse --ansi --error-format symplify",
+        "rector": "\"vendor/bin/rector\" process --config rector-ci.yaml",
         "changelog": [
-            "vendor/bin/changelog-linker dump-merges --in-categories --ansi",
-            "vendor/bin/changelog-linker link --ansi",
-            "vendor/bin/changelog-linker cleanup --ansi"
+            "\"vendor/bin/changelog-linker\" dump-merges --in-categories --ansi",
+            "\"vendor/bin/changelog-linker\" link --ansi",
+            "\"vendor/bin/changelog-linker\" cleanup --ansi"
```

Comment:
Why this? Should be reverted

Reply:
![cmd](https://user-images.githubusercontent.com/2391460/77131146-c0bc1380-6a84-11ea-8665-5be153dc59e8.png)

Reply:
There is trick for this to shorten it to part after `bin`.
But that breaks while copying the command and modifying it. So this it the final choice.